Source/Urho3D/CSharp/Managed/DerivedFromAttribute.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024-2024 the rbfx project.
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT> or the accompanying LICENSE file.
+
+namespace Urho3DNet
+{
+    /// <summary>
+    /// Add interface to the list of base types to allow <see cref="Node.GetDerivedComponent{T}"/> find component by the interface.
+    /// </summary>
+    [System.AttributeUsage(System.AttributeTargets.Interface)]
+    public sealed class DerivedFromAttribute : System.Attribute
```

Comment:
How are we supposed to use it?

Reply:
Developer can create interfaces and mark them with the attribute. Interfaces like IInteractable, IHoverable, IFirearm, ICharacter, IHealthManager, etc. Then multiple implementations of the interfaces allowed. For example ICharacter could be Player or NPC. IInteractable could be Door or KeyCard, etc.

Reply:
You'll see it soon in the thirdperson project template. I intent to extend it to more advanced base for a game.